src/System.Runtime.Intrinsics/ref/System.Runtime.Intrinsics.cs

```diff
@@ -0,0 +1,22 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+// ------------------------------------------------------------------------------
+// Changes to this file must follow the http://aka.ms/api-review process.
+// ------------------------------------------------------------------------------
+
+using System.Runtime.InteropServices;
+
+namespace System.Runtime.Intrinsics
+{
+    [StructLayout(LayoutKind.Sequential, Size = 16)]
+    public struct Vector128<T> where T : struct 
+    {
+        public static bool IsSupported { get { throw null; } }
```

Comment:
When is `Vector128<T>.IsSupported` going to return true?

Having IsSupported property on the vector types looks odd as I am thinking through it. Do we really need it?

Reply:
IMO it should be sufficient for Intel intrinsics to have IsSupported on classes implementing given ISA extension i.e. Avx or Avx2. However, in case of ARM SVE intrinsics we would have many different vector sizes available with this same intrinsics so it would be necessary to know which vector sizes are actually supported on given hardware. In short know it is not required but in future it will be requiered (providing ARM will implement SVE as currently described in beta documentation).

Reply:
If it is ARM SVE specific, it should be eventually under ARM SVE API set. Or in case it really needs to be here for ARM SVE, it should be added with the rest of ARM SVE.

Reply:
I'm not sure if it is required.

Given that the `VectorXXX<T>` types are completely opaque and don't expose fields, properties, or methods themselves. It should be sufficient to only have `IsSupported` checks on the classes that expose the intrinsics (`Sse`, `Sse2`, `Avx`, etc). If `Sse` is supported and it exposes a method that uses `Vector128<T>`, then `Vector128<T>` is implicitly supported. Likewise, if `Avx2` is implemented and uses `Vector512<T>`, then it is implicitly supported whenever `Avx2.IsSupported` returns true.

Reply:
Originally, we design the `Vector128<T>.IsSupported` for conversion functions between the existing SIMD type (`Vector<T>`) and `Vector128<T>` and `Vector256<T>`. But, currently, we do not have them, so I will remove `Vector128<T>.IsSupported`.